README.md

```diff
@@ -150,18 +150,22 @@ The image is the main element. It defines the runtime stage of the Dockerfile:
 | `user`           | String?          | The runtime user (default `1000`) |
 | `workdir`        | String?          | The runtime work directory    |
 | `envs`           | Map<String, String>? | The runtime environment variables |
+| `env`            | Map<String, String>? | `envs` alias              |
 | `artifacts`      | [Artifact](#artifact)[]? | Defines artifacts to copy from builders |
 | `adds`           | String[]?        | Paths of elements to add at build time to the workdir |
+| `add`            | String[]?        | `adds` alias                  |
 | `root`           | [Root](#root)?   | Actions made using the `root` user |
 | `script`         | String[]?        | Script commands to execute    |
 | `run`            | String?          | `script` alias                |
 | `caches`         | String[]?        | Paths in the image stage to cache during the `script` execution. Be careful when using caches because the cached directory is not present after the script execution |
+| `cache`          | String[]?        | `caches` alias                |
```

Comment:
Maybe showing aliases in an other column can be better than adding more lines in the tab.
It can be easier to read because it will have less lines that show each functionnality we can set in our dofigen with prefered value in first look. Other possibilities in an Alias column if we want.